jetstream/kv.ts

```diff
@@ -917,6 +920,10 @@ export class KvStatusImpl implements KvStatus {
   get size(): number {
     return this.si.state.bytes;
   }
+
+  get metadata(): Record<string, string> | undefined {
+    return this.si.config.metadata;
```

Comment:
Sure you don't want to default to an empty object when undefined? less cognitive overload on the caller.

Reply:
Also looks like object store does return ?? empty.

Reply:
you are absolutely correct